File: LeetcodeDaily/26.removeDuplicatesfromsortedArray.py
class Solution:
    def removeDuplicates(self, nums: list[int]) -> int:

        res=1
        for i in range(1,len(nums)):
            if nums[i]!=nums[i-1]:
                nums[res]=nums[i]
                res+=1
                # print(nums)

        return res

File: LeetcodeDaily/X/26. Remove Duplicates from Sorted Array.java
//Input: nums = [1,1,2]
//Output: 2, nums = [1,2,_]

import java.util.HashMap;
import java.util.Map;

class Solution {
    public int removeDuplicates(int[] nums) {
        //Remove Duplicate Elements
        HashMap<Integer,Integer> map=new HashMap();
        int temp=0;
        for(int i=0;i<nums.length-1;i++){
            temp=nums[i+1];
            if(nums[i]!=temp){
                map.put(nums[i],i);
            }
        }//Convert HashMap to arr

        return map.values(Integer);
    }
}